include/safe/var.hpp

```diff
@@ -96,11 +141,15 @@ namespace safe {
         }
     }
 
-    [[nodiscard]] SAFE_INLINE constexpr auto operator+(Var auto lhs, Var auto rhs) {
+    template<typename L, typename R>
+    requires (at_least_one_var<L, R>)
```

Comment:
**thought**: With the concept, this could be written
```cpp
template <typename L, at_least_one_var<L> R>
```
but I think I prefer the symmetry of the explicit requirement.

Reply:
I prefer the symmetric option as well.